README.md

```diff
@@ -1,29 +1,32 @@
-[![Build Status](https://travis-ci.org/clemahieu/raiblocks.svg?branch=master)](https://travis-ci.org/clemahieu/raiblocks) 
-___
-
+RaiBlocks  
```

Comment:
Use `# RaiBlocks` here rather than `====` for consistency?

Consider also blank newline after all headers.